integrations/jina/src/haystack_integrations/components/embedders/jina/text_embedder.py

```diff
@@ -48,22 +48,22 @@ def __init__(
         :param suffix: A string to add to the end of each text.
         """
 
-        api_key = api_key or os.environ.get("JINA_API_KEY")
-        # we check whether api_key is None or an empty string
-        if not api_key:
+        resolved_api_key = api_key.resolve_value()
+        if resolved_api_key is None:
             msg = (
-                "JinaTextEmbedder expects an API key. "
+                "JinaDocumentEmbedder expects an API key. "
```

Comment:
Needs to be changed back to `JinaTextEmbedder`

Reply:
fixed. Thanks!